### DISCUSSION TITLE
Onboarding MRs for mirrors should warn users that Renovate will skip it

### DISCUSSION THREAD

**Discussion by colinodell** [renovatebot/renovate#26168] [Suggest an Idea]
### Tell us more.

We had a user add Renovate to their repository, which happens to be configured as a mirror.  Renovate opened the usual onboarding MR, showing all of the detected package files, configuration summary, etc.  However, after merging that MR, the repo owner was confused why none of the promised MRs were being opened.

The answer was found in the logs:

```
Repository is a mirror - skipping
```

It turns out that [Renovate skips mirrors by default](https://docs.renovatebot.com/self-hosted-configuration/#includemirrors) but didn't inform the repo owner about this.

It would be great if the onboarding step could provide a prominent warning to the user if their repository would be skipped.

**Comment by rarkins** [comment]
Ideally Renovate wouldn't skip it though if it can determine their intent? Can you describe in more detail, e.g. you have includeMirrors=true in bot config so it's autodiscovered, but then Renovate still skips it during initRepo?

**Reply by colinodell** [reply]
includeMirrors is not explicitly set, so I'm pretty sure we're using the default of `false`.

We use Renovate CE, but I think I manually invoked `./node_modules/.bin/renovate org/repo` to figure out why Renovate "wasn't working" as expected (I should have checked the logs instead), and this seemed to have created the onboarding MR.  So perhaps autodiscovery _is_ working properly, but manually running Renovate against an explicit repository somehow avoids the skip logic?  (I haven't had time to confirm any of this yet)
